Pages/ManagePotentialVolunteer.razor.cs

```diff
@@ -58,7 +58,25 @@ public IEnumerable<ICharityEvent> GetEventsOfThisVolunteer()
 
         public IEnumerable<ICharityEvent> GetEventsNotOfThisVolunteer()
         {
-            return m_karmaContext.Events.Include(p => p.Volunteers);
+            return m_karmaContext.Events.Include(p => p.Volunteers).Where(p => p.ManagerId == CurrentUserId && p.MaxVolunteers > p.Volunteers.Count);
+        }
+
+        public IEnumerable<ICharityEvent> GetHighPriorityEvents()
```

Comment:
If I'm correct I saw 3 exact same methods. I think it would be better to have some class which has this method and reuse it

Reply:
Did it with 2 repeating methods.